wso2bps/kubernetes/wso2bps-worker-controller.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: wso2bps-worker
+  labels:
+    name: wso2bps-worker
+spec:
+  replicas: 1
+  selector:
+    name: wso2bps-worker
+  template:
+    metadata:
+      labels:
+        name: wso2bps-worker
+    spec:
+      containers:
+      - name: wso2bps-worker
+        image: wso2/bps-worker-3.5.0:1.0.0
+        ports:
+        - containerPort: 8243
```

Comment:
Shall we also include a container port for 8280 and mention the protocol as TCP?